src/Node/Render.php

```diff
@@ -69,43 +76,50 @@ public static function getEnvironment(): \Twig_Environment {
    */
   protected function addGetTemplate(Twig_Compiler $compiler) {
     $this->setEnvironment($compiler->getEnvironment());
-    $compiler->raw('$handles = (array) ')->subcompile($this->getNode('expr'))->raw(';');
+    $compiler->raw('$handles = (array) ')
+      ->subcompile($this->getNode('expr'))
+      ->raw(';');
```

Comment:
Can you revert all the coding style changes, please? You can propose them in a separate PR, but for this one they are out of scope.

Reply:
Ok, thanks

Reply:
@sun coding standards changes are reverted now https://github.com/fathimasmat/twig_fractal/commit/6802008b4fe713ab480c2ba34c70e2d73eb32842